modules/ipi-install-troubleshooting-misc-issues.adoc

```diff
@@ -77,6 +77,31 @@ $ kubectl -n openshift-network-operator logs -l "name=network-operator"
 +
 On high availability clusters with three or more control plane (master) nodes, the Operator will perform leader election and all other Operators will sleep. For additional details, see https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md[Troubleshooting].
 
+== Addressing the `No disk found with matching rootDeviceHints` error
+
+After the deployment of a cluster you might receive the following error:
+
+----
+`No disk found with matching rootDeviceHints`
+----
+
+To address the "No disk found with matching rootDeviceHints" error, a temporary workaround is to change the `rootDeviceHints` to `minSizeGigabytes: 300`.
+
+After changing the `rootDeviceHints`, boot the CoreOS and then verify the disk information by using the command:
+
+[source]
+----
+$ udevadm info /dev/sda
+----
+
+If you are using DL360 Gen 10 servers, be aware that they have an SD-card slot which might be assigned the `/dev/sda` device name. If no SD card is present in the server, it can cause conflicts. Ensure that the SD card slot is disabled in the server's BIOS settings.
```

Comment:
```suggestion
If you are using DL360 Gen 10 servers, be aware that they have an SD-card slot that might be assigned the `/dev/sda` device name. If no SD card is present in the server, it can cause conflicts. Ensure that the SD card slot is disabled in the server's BIOS settings.
```
Use "that" instead of "which" for restrictive clauses. Reference: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-with-nonrestrictive-clauses-and-phrases